deploy/kustomize/base/subscription.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: hco-operatorhub
+  namespace: ${TARGET_NAMESPACE}
+spec:
+  source: ${CSC_SOURCE}
+  sourceNamespace: ${MARKETPLACE_NAMESPACE}
+  name: kubevirt-hyperconverged
+  startingCSV: kubevirt-hyperconverged-operator.v${HCO_VERSION}
+  channel: "${HCO_CHANNEL}"
+  installPlanApproval: Automatic
+  ${SUBSCRIPTION_CONFIG}
```

Comment:
Along the lines of being more prescriptive, I would think we would define as little as necessary on the Spec of the Subscription. My understanding is the essentials are the name and channel. Including the `installPlanApproval` seems reasonable (even though it is the default) because we want consumers to allow automatic updates.

Reply:
Maybe overlays are the way to add these to the spec when appropriate?

Reply:
Thanks  for commenting, David.
In other words, you suggest to move the fields that have high chance to be changed from default values, into overlays with `patchesStrategicMerge` that they'll overwrite the defaults?
e.g. in subscription, that would be name and channel, while the others will be editable on base?